xonsh/xoreutils/uname.py

```diff
@@ -0,0 +1,209 @@
+import platform
+import sys
+from xonsh.xoreutils.util import arg_handler
+
+
+def uname(args, stdin, stdout, stderr):
+    """
+    Print certain system information.
+
+    :param args: Arguments like -a
+    :type args: list or None
+    :param stdin: The standard input
+    :type stdin: sys.stdin
+    :param stdout: The standard output
+    :type stdout: sys.stdout
+    :param stderr: The standard output
+    :type stderr: sys.stderr
```

Comment:
Please use numpydoc style docstrings, like elsewhere in xonsh. https://numpydoc.readthedocs.io/en/latest/format.html